content/en/platform/corda/5.2/developing-applications/api/ledger/utxo-ledger/vault-queries.md

```diff
@@ -389,8 +396,8 @@ utxoLedgerService.query("DUMMY_CUSTOM_QUERY", Integer.class)
 Provide the name of the query (in this case `DUMMY_CUSTOM_QUERY`) and the return type. Since the result set is collected into an integer in the complex query example, use `Int` (or `Integer` in Java).
 Before executing, define the following:
 
-* How many results each page of the query should return (`setLimit`), default Int.MAX (2,147,483,647).
-* Define named parameters that are in the query and the actual value for them. {{< note >}} All parameters must be defined, otherwise the execution will fail. (`setParameter` or `setParameters`) {{</ note >}}
+* How many results each page of the query should return (`setLimit`), the default value is Int.MAX (2,147,483,647).
+* Define named parameters that are in the query and the actual value for them. ALl parameters must be defined, otherwise the execution will fail. (`setParameter` or `setParameters`).
```

Comment:
```suggestion
* Define named parameters that are in the query and the actual value for them (`setParameter` or `setParameters`). All parameters must be defined, otherwise the execution will fail.
```